packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -613,36 +616,38 @@ export default class SpecPreview extends GlimmerComponent<Signature> {
     return this.realm.canWrite(this.operatorModeStateService.realmURL.href);
   }
 
-  private search = getSearch(
-    this,
-    () => this.specQuery,
-    () => this.realms,
-    { isLive: true, isAutoSaved: true },
-  );
+  private makeSearch = () => {
+    this.search = this.getCards(
+      this,
+      () => this.specQuery,
+      () => this.realms,
+      { isLive: true },
+    ) as ReturnType<getCards<Spec>>;
```

Comment:
This search is run after a realm indexing event is received in code mode. specifically this is the only search that is running as code is being changed in the code editor. (and it is one query per realm per realm index event received)